metadata/SSCWeb/SSCWeb2HAPI.js

```diff
@@ -104,15 +142,18 @@ function makeHAPI(jsonraw,cb) {
 			catalog[i]["info"]["parameters"][j]["description"] = paraminfo[2].replace(/"/g,"");
 			catalog[i]["info"]["parameters"][j]["units"] = paraminfo[3].replace(/"/g,"") || "dimensionless";
 			catalog[i]["info"]["parameters"][j]["fill"] = paraminfo[4].replace(/"/g,"") || null;
+
+			
 			var type = paraminfo[5].replace(/"/g,"");
-			//console.log(catalog[i]["info"]["parameters"][j]["name"],type,parseInt(type.replace(/%|s/,"")));
-			if (/f$/.test(type)) {
+			//console.log(catalog[i]["info"]["parameters"][j]["type"],type,parseInt(type.replace(/%|s/,"")));
+
+			if (type.substring(type.length-2,type.length-1) == "f") {
```

Comment:
The time parameter was not being added to the catalog entries. Hence tweaked it by replacing regex with simple substring to make it work.